### PR TITLE
[metascraper-clearbit] expose some package options

### LINKED ISSUE #196: [metascraper-clearbit] 1000 ms timeout should be configurable
### Prerequisites

- [x] I'm using the last version.
- [x] My node version is the same as declared as `package.json`.

### Subject of the issue

I expected `metascraper-clearbit` to properly fetch the logo for a given URL, but it intermittently returns `null` for the logo instead. I found this is because I'm using the library from Japan, and the timeout for the clearbit API request is not configurable, set to 1000. When I edited the library directly in `node_modules/`, it suddenly started working far more reliably.

On a related note, there ought to be some way to detect errors, they're useful to know - I was confused for some time because of this.

### Steps to reproduce

Get on a high-latency connection and try to use `metascraper-clearbit` (i.e., with the demo code). You can simulate this by editing `metascraper-clearbit` to have an unreasonably short timeout in the `got` request, and then rather than squelching errors in the `try/catch` in the code, log them.

### Expected behaviour

`logo` should get a Clearbit logo, because running that URL in [the Clearbit Logo API demo page](https://clearbit.com/logo) does produce one

### Actual behaviour

You get `null` instead, and if you do the above, then you'll see a TIMEOUT error.

### LINKED ISSUE #197: Contributing: not entirely clear how to set up, dev setup seems broken?
### Prerequisites

- [x] I'm using the last version.
- [x] My node version is the same as declared as `package.json`. (It just says `>=8`, I'm on `12.4.0`)

### Subject of the issue

I'm taking guesses at how to set up the repo for development (`npm install`?), but I don't know what exactly I'm supposed to do so I don't feel confident making changes. `CONTRIBUTING.md` pretty light on details.

- `npm run-script install` was stuck installing forever (like more than 20 minutes bootstrappign dependencies, stuck on `install dependencies: verb installed leaf metascraper` forever) so I gave up on it finishing
- lint fails when I run `npm test`, saying that `metascraper-helpers` and a few other packages are not following Standard JS
- I don't know how to run tests for a specific subpackage (`metascraper-clearbit`) so I just have to run them all
- when I do tests with `lerna exec npm run test` to get around the pretest `lint` that keeps failing, those fail too because individual packages I didn't edit cant find `@metascraper/helpers`
-  the husky `git-commit` hook is failing because of some invalid values passed to a library called `lint-staged`, but I don't know what any of this is
- docs are very light on things like, say, the `metascraper-helpers` lib that a lot of stuff seems to rely on

<details><summary>git commit hook error</summary>

```
husky > pre-commit (node v12.4.0)
git push -u Could not parse lint-staged config.

        Error: ● Validation Error:

Invalid value for 'linters'.

Advanced configuration has been deprecated. For more info, please visit: https://github.com/okonet/lint-staged.

Configured value is: {'*.js': ['prettier-standard', 'git add'], 'package.json': ['finepack', 'git add']}

Please refer to https://github.com/okonet/lint-staged#configuration for more information...
● Validation Error:

Invalid value for 'linters'.

Should be a string, a function, or an array of strings and functions.

Configured value is: {'*.js': ['prettier-standard', 'git add'], 'package.json': ['finepack', 'git add']}

Please refer to https://github.com/okonet/lint-staged#configuration for more information...
● Validation Error:

Invalid value for 'ignore'.

Advanced configuration has been deprecated. For more info, please visit: https://github.com/okonet/lint-staged.

Configured value is: ['static/*.{js, css}']

Please refer to https://github.com/okonet/lint-staged#configuration for more information...

Please make sure you have created it correctly.
See https://github.com/okonet/lint-staged#configuration.
husky > pre-commit hook failed (add --no-verify to bypass)
```

</details>

Would be nice if maintainers included a bit more detail on how I'm supposed to configure things to develop on this project!

### LINKED ISSUE #198: [metascraper-clearbit] Options not implemented?
### Prerequisites

- [x] I'm using the last version.
- [x] My node version is the same as declared as `package.json`.

### Subject of the issue

I think the existing documented fields of `size` `greyscale`, `format` aren't actually implemented - no reference to them in the source nor tests.

### PR DESCRIPTION
**Pull request by osdiab**
Closes #196
Closes #197
Closes #198

Beware: I'm not actually sure that this works since I can't run the tests locally and I also don't know how stuff actually gets passed to metascraper rules, being the first one i've edited; #197 for details. Therefore this doesn't have updated snapshots.

### CODE REVIEW

**Comment by Kikobeats** [comment]
@osdiab No problem, thanks for the effort. I made a commit doing some tweaks; now, the package supports any option related to Clearbit or the HTTP layer 🙂

File: packages/metascraper-clearbit/README.md [hunk 1]
```diff
@@ -13,30 +13,27 @@ $ npm install metascraper-clearbit --save
 
 ## API
 
-### clearbitLogo([options])
+### netascraper-clearbit([options])
```
Comment (osdiab) [inline packages/metascraper-clearbit/README.md]: Typo?

Comment (Kikobeats) [reply packages/metascraper-clearbit/README.md]: 👍

**Comment by Kikobeats** [comment]
Released at `v5.6.4` 🎉

**Pull request merged**